templates/phaser/packages/client/src/mud/setupNetwork.ts

```diff
@@ -21,13 +39,23 @@ export async function setupNetwork() {
 
   const publicClient = createPublicClient(clientOptions);
 
+  /*
+   * Create A temporary wallet and a viem client for it
```

Comment:
```suggestion
   * Create a temporary wallet and a viem client for it
```
nit (this needs updating everywhere)